models/iaf_psc_alpha.h

```diff
@@ -34,89 +34,122 @@
-   ``tau_syn_in``, respectively, to avoid numerical instabilities.
+   NEST uses exact integration [1]_, [2]_ to integrate subthreshold membrane
+   dynamics with maximum precision; see also [3]_.
+
+   If :math:`\tau_\text{m}\approx \tau_{\text{syn, ex}}` or
+   :math:`\tau_\text{m}\approx \tau_{\text{syn, in}}`, the model will
+   numerically behave as if :math:`\tau_\text{m} = \tau_{\text{syn, ex}}` or
+   :math:`\tau_\text{m} = \tau_{\text{syn, in}}`, respectively, to avoid
+   numerical instabilities.
 
    For implementation details see the
    `IAF Integration Singularity notebook <../model_details/IAF_Integration_Singularity.ipynb>`_.
 
-See also [3]_.
 
 Parameters
 ++++++++++
 
 The following parameters can be set in the status dictionary.
 
-=========== ======  ==========================================================
- V_m        mV      Membrane potential
- E_L        mV      Resting membrane potential
- C_m        pF      Capacity of the membrane
- tau_m      ms      Membrane time constant
- t_ref      ms      Duration of refractory period
- V_th       mV      Spike threshold
- V_reset    mV      Reset potential of the membrane
- tau_syn_ex ms      Rise time of the excitatory synaptic alpha function
- tau_syn_in ms      Rise time of the inhibitory synaptic alpha function
- I_e        pA      Constant input current
- V_min      mV      Absolute lower value for the membrane potenial
-=========== ======  ==========================================================
+=============== ======== =============================== ========================================================================
+**Parameter**   **Unit** **Math equivalent**             **Description**
+=============== ======== =============================== ========================================================================
+ ``V_m``         mV       :math:`V_{\text{m}}`           Membrane potential
+ ``E_L``         mV       :math:`E_\text{L}`             Resting membrane potential
+ ``C_m``         pF       :math:`C_{\text{m}}`           Capacity of the membrane
+ ``tau_m``       ms       :math:`\tau_{\text{m}}`        Membrane time constant
+ ``t_ref``       ms       :math:`t_{\text{ref}}`         Duration of refractory period
+ ``V_th``        mV       :math:`V_{\text{th}}`          Spike threshold
+ ``V_reset``     mV       :math:`V_{\text{reset}}`       Reset potential of the membrane
+ ``tau_syn_ex``  ms       :math:`\tau_{\text{syn, ex}}`  Rise time of the excitatory synaptic alpha function
+ ``tau_syn_in``  ms       :math:`\tau_{\text{syn, in}}`  Rise time of the inhibitory synaptic alpha function
+ ``I_e``         pA       :math:`I_\text{e}`             Constant input current
+ ``V_min``       mV       :math:`V_{\text{min}}`         Absolute lower value for the membrane potenial (default :math:`-\infty`)
+=============== ======== =============================== ========================================================================
```

Comment:
While I think the table is very neat, a tabular format might actually become quite restrictive for describing parameters due to the limited horizontal space. I suggest to instead use a list format akin to the parameter section in Python documentation. We already use that here https://nest-simulator.readthedocs.io/en/latest/models/poisson_generator.html#parameters. I also think that we should provide the default value of the parameters in the documentation.

Reply:
Also in view of @jougs comment below, I would not want to change this now.